Workflow/Tests/HostContextTests.swift

```diff
@@ -0,0 +1,23 @@
+import XCTest
+
+@testable import Workflow
+
+final class HostContextTests: XCTestCase {
+    func test_conditional_debug_info_no_debugger() {
+        let subject = HostContext.testing(debugger: nil)
+        subject.ifDebuggerEnabled {
+            XCTFail("should not be called")
+        }
+    }
+
+    func test_conditional_debug_info_with_debugger() {
+        let subject = HostContext.testing(debugger: TestDebugger())
+        let expectaiton = expectation(description: "debugger block invoked")
```

Comment:
```suggestion
        let expectation = expectation(description: "debugger block invoked")
```
Did you know you can turn on spellcheck in Xcode?

Reply:
ha... yes i do, and (perhaps surprisingly) do in fact have that setting enabled. guess i was not paying very close attention...